File: dynamic_proxy_hook/src/main/java/com/viseator/pluginframeworkdemo/TargetActivity.java
package com.viseator.pluginframeworkdemo;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;

public class TargetActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_target);
        Log.d("@vir", "started");
        setTitle(R.string.target_title);
    }
}

File: undecleared_activity/src/main/java/com/viseator/undeclearedactivity/TargetActivity.kt
package com.viseator.undeclearedactivity

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.util.Log

/**
 * Created on 2018/7/24.
 * wudi.viseator@bytedance.com
 */

class TargetActivity : AppCompatActivity() {
    init {
        Log.d("@vir", "Target Activity inited")
    }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.target_activity_layout)
    }
}